frontend/src/components/commons/SideNavigation.jsx

```diff
@@ -65,30 +72,41 @@ class SideNavigation extends Component {
   render() {
     return (
       <div className="side-nav-grid">
-        <div className="side-nav-grid-buttons-cell" style={styles.buttonList}>
-          {this.props.navSpecs.map(tab => (
-            <div key={tab.id}>
-              {tab.id === this.state.currentNode && (
-                <button
-                  className="btn-primary"
-                  style={{ ...styles.button, ...styles.primaryButton }}
-                  onClick={() => this.changeNode(tab.id)}
-                >
-                  {tab.text}
-                </button>
-              )}
-              {tab.id !== this.state.currentNode && (
-                <button
-                  className="btn-secondary"
-                  style={{ ...styles.button, ...styles.secondaryButton }}
-                  onClick={() => this.changeNode(tab.id)}
-                >
-                  {tab.text}
-                </button>
-              )}
-            </div>
-          ))}
-        </div>
+        <nav
+          className="side-nav-grid-buttons-cell"
+          style={styles.buttonList}
+          role="dialog"
```

Comment:
Wouldn't this be role nav too?

Reply:
role nav or navigation are not detected the narrator at all. Instead it just reads the buttons/tab/etc like a bulleted list. Only dialog seems to work in any browser for some reason.

Reply:
Is this a well documented work around? Soon we'll be adding a dialog, and having multiple items tagged as dialog is going to mess things up. 

If it is, we need to add a comment to this file, otherwise, is there any other way to control focus and use the correct labels?

Reply:
@fnormand01 would know if there is. He found the work around for this. It is already implemented for the main nav on master.

Reply:
I'm going to document this as something we need to come back to, and we can merge this PR.